README.md

```diff
@@ -1,36 +1,10 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src=" " alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Great job 👍 using [Microverse template](https://github.com/microverseinc/readme-template) for your README.md however as requested by the previous reviewer please 🙏  make sure to customise this template by adding descriptions to each section, reflecting your project. Also make sure to take care of the following:

       - Please remove Note from the LICENCE section of the README. Note is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it
       - The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.
       - Please remove the Microverse logo from your readme. Optionally, you can replace with your own logo if you want.
       - Please remove the default comment like
       
            > Give credit to everyone who inspired your codebase.  
           > Write a message to encourage readers to support your project  
           > Describe 1 - 3 features you will add to the project.  
          
          ,etc from your README. 
          They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README